lib/text.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
```

Comment:
I saw this gem was missing from your Gemfile, it'd be good to add it so that anyone running 'bundle' will get it downloaded